docs/reference/cli.md

```
@@ -64,6 +66,23 @@ Images are stored in `~/.smolvm/images`. To keep them somewhere else, set the `S
+```bash
+# bash — add to ~/.bashrc
+eval "$(smolvm completion bash)"
+
+# zsh — add to ~/.zshrc
+eval "$(smolvm completion zsh)"
+
+# fish — write once to the completions folder
+smolvm completion fish > ~/.config/fish/completions/smolvm.fish
+```
+
+Open a new shell afterward, then type `smolvm sandbox ssh ` and press Tab to complete a sandbox name.
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)